torch/csrc/Exceptions.h

```diff
@@ -127,6 +133,7 @@ static inline void PyErr_SetString(PyObject* type, const std::string& message) {
 #define END_HANDLE_TH_ERRORS END_HANDLE_TH_ERRORS_RET(nullptr)
 
 extern PyObject *THPException_FatalError;
```

Comment:
It looks like THPException_FatalError never gets used anywhere so it's not surprising that this pattern doesn't actually work